bot/constants.py

```diff
@@ -4,11 +4,27 @@
 
 import yaml
 
+
 # env vars
 PREFIX = os.getenv("PREFIX") or "!"
 TOKEN = os.getenv("TOKEN")
 BOT_REPO_URL = "https://github.com/gurkult/gurkbot"
 
+
+class Roles(typing.NamedTuple):
```

Comment:
```suggestion
class Roles(NamedTuple):
```
Also move this class below `Channels`